tsconfig.json

```diff
@@ -1,17 +1,34 @@
 {
```

Comment:
@NikhilSharma03 Great work!! but wish this file unchanged. 
## Heres my explanation why.
create-react-app(CRA) seems to change tsconfig.json! but changed tsconfig.json :  `{"module": "esnext", ~~}` it seems to be a problem for backend application to fail. [Link]( https://stackoverflow.com/questions/53794875/how-to-configure-react-script-so-that-it-doesnt-override-tsconfig-json-on-star)
## Heres my idea
1. I'm thinking splitting Development process to FE & BE! so each of stuff's dependency doesn't overrides each other 
2. ejecting CRA. I don't like this method but it can be a solution!

Reply:
Oops! I think I accidentally added the ts.config file too. :sweat_smile: